README.md

```diff
@@ -54,3 +55,51 @@ This repository implments tests using [pytest](https://docs.pytest.org/), the te
 
 [^env_vars]: [For linux](https://www.gnu.org/software/bash/manual/bash.html#Environment) you can either prepend as in `NAME=value pipenv ...` or use `export NAME=value`, [for windows](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/set_1) you can use `set`.
 [^user_config]: A more detailed documentation on [the user configuration file can be found here](docs/user_configuration_file.md).
+
+## AWS Packing
+Since some services from infin-neat-core code runs at AWS cloud services, it's necessary to properly pack the code to upload it using the web interface, AWS CLI, or AWS CDK. 
+
+For that, on specific script has been created and can be run using pipenv: 
+```
+pipenv run aws_pack
+```
+### Requirements:
+
+#### **Install Docker**: 
+Since in the packing processing Docker is used, it's necessary to install it. For installing Docker, the official documentation from Docker can be followed for [Linux](https://docs.docker.com/desktop/install/linux-install/), [Windows](https://docs.docker.com/desktop/install/windows-install/), and [MAC](https://docs.docker.com/desktop/install/mac-install/), 
+
+
+#### **Running Docker Rootless**: 
+
+Since the script uses **Docker** without root privileges, some extra steps are necessary to run the Docker daemon as a non-root user (check [Rootless mode](https://docs.docker.com/engine/security/rootless/) for more details). For it, follow the following steps:
```

Comment:
It would be beneficial to say why the aws_lambda_pack command needs to run Docker without root privileges. I'd add something like:

The `aws_lambda_pack` command uses Docker without privileges, <your reason here, runs without password, etc..>. Therefore, extra steps are needed to run docker daemon as a non-root user.